R/derive_param_bor.R

```diff
@@ -121,14 +121,14 @@
 #'
 #' @param set_values_to New columns to set
 #'
-#'   A named list returned by `vars()` defining the columns to be set for the
-#'   new parameter, e.g. `vars(PARAMCD = "BOR", PARAM = "Best Overall
+#'   A named list returned by `exprs()` defining the columns to be set for the
+#'   new parameter, e.g. `exprs(PARAMCD = "BOR", PARAM = "Best Overall
 #'   Response")` is expected. The values must be symbols, character strings,
 #'   numeric values, or `NA`.
 #'
 #' @param subject_keys Columns to uniquely identify a subject
 #'
-#'   A list of symbols created using `vars()`.
+#'   A list of symbols created using `exprs()`.
 #'
 #'   *Permitted Values:* an `vars` object
```

Comment:
Should this instead be: an `exprs` object ?

Reply:
Yes, good catch! I have updated it.